e2e/app.po.ts

```diff
@@ -1,7 +1,35 @@
 import { browser, by, element } from 'protractor';
 
 export class AppPage {
+	private _LOGIN_COMPONENT_SELECTOR = 'ansyn-login';
+	loginComponent = element(by.css(this._LOGIN_COMPONENT_SELECTOR));
+
+	private _usernameInput = element(by.id('username'));
+	private _passwordInput = element(by.id('password'));
+	private _CORRECT_USERNAME = 'admin@ansyn.com';
+	private _CORRECT_PASSWORD = '123789';
```

Comment:
don't expose password in code, talk with @liorze about a solution

Reply:
Waiting for @liorze to complete branch to enable use of environment variables